bats/tests/helpers/os.bash

```diff
@@ -84,16 +84,26 @@ sudo_needs_password() {
 
 supports_vz_emulation() {
     if is_macos; then
+        if [[ -n ${_RD_SUPPORTS_VZ_EMULATION:-} ]] || load_var _RD_SUPPORTS_VZ_EMULATION; then
+            run is_true "$_RD_SUPPORTS_VZ_EMULATION"
+            return "$status"
+        fi
         local version
         version=$(semver "$(/usr/bin/sw_vers -productVersion)")
         trace "macOS version is $version"
         if semver_gte "$version" 13.3.0; then
+            _RD_SUPPORTS_VZ_EMULATION=1
```

Comment:
It doesn't really matter, as the code is correct as-is, but I noticed that we are using `true` and `false` in all the `using_*` predicates, so if you make any other changes, maybe use the same here instead of `1` and `0`?